README.md

```diff
@@ -57,3 +86,7 @@ Run tests with:
 ```sh
 rspec
```

Comment:
We are running tests on Travis with bundle exec rake. Considering I'm a noob, does this make a difference?